Automattic/wp-calypso
# fix: tweaked the duration and font of the wooexpress setup messages

Related to #73872

## Proposed Changes

* Adjusted the timings and messages and subtitle font for the WooExpress setup stepper

## Testing Instructions

- Set up the repo and load http://calypso.localhost:3000/setup/wooexpress
- Or use the live link
- Observe that the loading titles should roughly correspond to:
    1. Woo! We're creating your store
    2. Building the foundations
    3. Organizing the stock room
    4. Applying the finishing touches
    5. Turning on the lights (may or may not see this depending on how long it takes to create the site)
    6. Opening the doors (may or may not see this depending on how long it takes to create the site)

Should look like this:


https://user-images.githubusercontent.com/27843274/222679870-a20b05f3-b3df-4045-b5a3-1b922b15af8c.mov



## Pre-merge Checklist
